### ISSUE TITLE
[AGW[HE][1.6.1] - he_enabled is not true after setting it in the NMS

### ISSUE THREAD

**Issue by sudhikan** [magma/magma#10338]
Prior to submitting an issue, check to see if one has already been created. If there is currently an open issue, add a thumbs-up emoji to identify that it is also affecting you.

### Your Environment

- **Version:** 1.6.1
- **Affected Component:** AGW
- **Affected Subcomponent:** Header Enrichment
- **Deployment Environment:** AWS (orc8r)bare-metal (AGW)

### Describe the Issue

HE is enabled in the NMS for *.fbmagma.ninja, however when grep-ing syslog for the `he_enabled` flag, we see it move from `true` to `false` stay there. Currently seeing this exclusively on `non-nat` setups. 

```
Nov 16 05:21:34 u-160-161-nonnat pipelined[1675021]: INFO:HeaderEnrichmentController:Header Enrichment app config: heConfig(he_proxy_port=1, he_enabled=True, encryption_enabled=False, encryption_algorithm=None, encryption_key=None, hash_function=None, encoding_type=None, uplink_port=2, gtp_port=32768)
Nov 16 05:54:40 u-160-161-nonnat pipelined[1681005]: INFO:HeaderEnrichmentController:Header Enrichment app config: heConfig(he_proxy_port=1, he_enabled=True, encryption_enabled=False, encryption_algorithm=None, encryption_key=None, hash_function=None, encoding_type=None, uplink_port=2, gtp_port=32768)
Nov 16 06:50:29 u-160-161-nonnat pipelined[1704370]: INFO:HeaderEnrichmentController:Header Enrichment app config: heConfig(he_proxy_port=1, he_enabled=True, encryption_enabled=False, encryption_algorithm=None, encryption_key=None, hash_function=None, encoding_type=None, uplink_port=2, gtp_port=32768)
Nov 16 06:51:45 u-160-161-nonnat pipelined[1706158]: INFO:HeaderEnrichmentController:Header Enrichment app config: heConfig(he_proxy_port=1, he_enabled=True, encryption_enabled=False, encryption_algorithm=None, encryption_key=None, hash_function=None, encoding_type=None, uplink_port=2, gtp_port=32768)
Nov 16 06:52:09 u-160-161-nonnat pipelined[1706715]: INFO:HeaderEnrichmentController:Header Enrichment app config: heConfig(he_proxy_port=1, he_enabled=True, encryption_enabled=False, encryption_algorithm=None, encryption_key=None, hash_function=None, encoding_type=None, uplink_port=2, gtp_port=32768)
Nov 16 06:56:23 u-160-161-nonnat pipelined[1713846]: INFO:HeaderEnrichmentController:Header Enrichment app config: heConfig(he_proxy_port=0, he_enabled=False, encryption_enabled=False, encryption_algorithm=None, encryption_key=None, hash_function=None, encoding_type=None, uplink_port=0, gtp_port=32768)
Nov 16 06:58:20 u-160-161-nonnat pipelined[1715533]: INFO:HeaderEnrichmentController:Header Enrichment app config: heConfig(he_proxy_port=0, he_enabled=False, encryption_enabled=False, encryption_algorithm=None, encryption_key=None, hash_function=None, encoding_type=None, uplink_port=0, gtp_port=32768)
Nov 16 07:14:30 u-160-161-nonnat pipelined[1725815]: INFO:HeaderEnrichmentController:Header Enrichment app config: heConfig(he_proxy_port=0, he_enabled=False, encryption_enabled=False, encryption_algorithm=None, encryption_key=None, hash_function=None, encoding_type=None, uplink_port=0, gtp_port=32768)
Nov 16 07:25:21 u-160-161-nonnat pipelined[1730941]: INFO:HeaderEnrichmentController:Header Enrichment app config: heConfig(he_proxy_port=0, he_enabled=False, encryption_enabled=False, encryption_algorithm=None, encryption_key=None, hash_function=None, encoding_type=None, uplink_port=0, gtp_port=32768)
Nov 16 07:30:07 u-160-161-nonnat pipelined[1736779]: INFO:HeaderEnrichmentController:Header Enrichment app config: heConfig(he_proxy_port=0, he_enabled=False, encryption_enabled=False, encryption_algorithm=None, encryption_key=None, hash_function=None, encoding_type=None, uplink_port=0, gtp_port=32768)
```

**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**
Add any other context about the problem here (e.g. logs).

**Comment by pshelar** [comment]
@sudhikan which command to you use to restart AGW services.

**Comment by sudhikan** [comment]
Hey @pshelar I used the following commands (in no particular order):

1. `service sctpd restart`
2. `config_stateless_agw.py flushall_redis` 
3. service magma@* stop && service magma@magmad start

**Comment by pshelar** [comment]
ok, It looks like the interface file is failing to create the port in some case. I will handle it in pipelineD.

**Comment by sudhikan** [comment]
Are we capturing this for 1.61?

**Comment by pshelar** [comment]
This issue would be fixed on 1.7 release.
workaround for this is as follows:

```
ifconfig proxy_port up
ovs-vsctl add-port gtp_br0 proxy_port
service magma@* stop
service magma@magmad start
```
